src/ocamlorg_frontend/pages/package_documentation.eml

```diff
@@ -15,7 +15,7 @@ Package_layout.render
 ~description:(Printf.sprintf "%s %s: %s" package.name package.version package.description)
 ~tab:Documentation
 ~package @@
-<button class="bg-primary-600  p-3 z-30 rounded-r-xl text-white shadow-md top-2/4 fixed md:hidden"
+<button class="bg-primary-600  p-3 z-30 rounded-r-xl text-white shadow-md top-2/4 fixed lg:hidden left-0"
```

Comment:
We should apply the same to problems and learn layout. Maybe worth extracting this to a component.